src/py42/sdk/queries/fileevents/filters/exposure_filter.py

```diff
@@ -130,3 +139,19 @@ class WindowTitle(FileEventFilterStringField):
     """
 
     _term = u"windowTitle"
+
+
+class RiskIndicatorMimeType(FileEventFilterStringField):
```

Comment:
you should use `QueryFilterBooleanField` here, see `Shared` for an example. No need to add any constants.

Reply:
Changed.